modules/nlu/src/views/full/intents/SidePanelSection.tsx

```diff
@@ -66,7 +66,7 @@ export const IntentSidePanelSection: FC<Props> = props => {
   const createIntent = async (sanitizedName: string, rawName: string) => {
```

Comment:
Can this method be called from this component ? Feel like its now impossible as as the `+` button is now one component higher